lessons/AI/ai-statistical-lang-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,452 @@
+- Click the "Collections" icon in the upper right corner.
+- From the drop-down menu that appears, select "tic-tac-toe".
+- Complete @printable-exercise{tic-tac-toe.adoc}
+}
+
+@teacher{
+Invite students to share their discoveries. We encourage you to allow students the opportunity to play with Soekia. There are many possible discoveries awaiting your students!}
+
+=== Synthesize
+
+@QandA{
+
+@Q{Does a statistical language model require language? How do you know?}
+
+@A{No. It just needs some sort of corpus. Natural language is absolutely not needed. Chess games, musical compositions, and tic-tac-toe games are all adequate!}
+}
+
+
+== Deep Dive: Soekia
+
+=== Overview
+
+Students engage with Soekia to discover some of the nuances of statistical language modeling.
+
+=== Launch
+
+Let's dig deeper into Soekia!
+
+@lesson-instruction{
+- Go to @link{http://Soekia.ch/GPT/?lang=en}
+- Complete the first section of @printable-exercise{soekia-intro.adoc}.
+- When you're done, let's do a quick survey: Raise your hand if your story was largely inspired by "Felicia and the Pot of Pinks".
+}
+
+@teacher{
+The vast majority of students will have a story that is primarily sourced from "Felicia and the Pot of Pinks". On the next section of the worksheet, students will discover exactly _why_ this is the case. Feel free to use this mystery as incentive to move on to the next section of the page!
+}
+
+@lesson-instruction{
+- Complete the second section of @printable-exercise{soekia-intro.adoc}.
+}
+
+@QandA{
+@Q{Why were so many of our initial stories all about Felicia and the Pot of Pinks?}
+@A{The green bar indicates how closely the document matches the box on the "Generate Text" panel. The story "Felicia and the Pot of Pinks" includes the word "tale" once, "fairy" four times, and the word "me" more than a dozen times. With these frequencies, it is a much closer match to "Write me a fairy tale" than any of the other documents.}
+}
+
+Let's review what we have done so far:
+
+- We have interacted Soekia's text generation panel. With modern AI, the text generating interface is the only element that we are privy to. Unlike the AI we use daily, Soekia allowed us to glimpse which words and phrases came from which sources.
+
+- We have also peeked at Soekia's documents panel, or corpus. This is a critical feature of all text-generating AI, but ordinarily, it is hidden from us. Soekia also revealed to us the level of alignment between each document and what _we_ typed into the box on the "Generate Text" panel.
+
+Let's explore the two remaining panels!
+
+@lesson-instruction{
+- Turn to @printable-exercise{soekia-closer-look.adoc}.
+- Be prepared to share your responses with the class.
+}
+
+@ifnotslide{
+@teacher{
+As students are working, you can share the three tips, below.
+}
+}
+
+@ifslide{
+Advance to the next slide for student-facing tips on navigating Soekia.
+}
+
+
+@slidebreak
+
+If you feel overwhelmed as you work, here are some tips:
+
```

Comment:
To be honest, perhaps it's because it's the end of a long day, but I think it's more likely because I've been so over commanded about where i need to be by directions that are incomplete with so little time on an open-ended task of interest to me, that I am not feeling curious at all.